.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
+* @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
I'm fine if @hillaryj or @mcdonnnj wants to whittle down this list.  I know for a fact that @jsf9k is not at all familiar with this repository.